app/jobs/reports/monthly_key_metrics_report.rb

```diff
@@ -58,8 +58,8 @@ def reports
         account_reuse_report.account_reuse_emailable_report,
         account_reuse_report.total_identities_emailable_report,
         monthly_proofing_report.document_upload_proofing_emailable_report,
-        # Number of applications using Login (separated by auth / IdV) - LG-11154
-        # Number of agencies using Login - LG-11155
+        agency_and_sp_report.agency_and_sp_emailable_report,
+        # Fiscal year active users, sum and split - LG-10816
```

Comment:
I think I took the wrong side in part of this merge conflict. This line should not be changed.